src/plugins/data_table/filter_view.js

```diff
@@ -85,6 +85,32 @@ var DataTableFilter = (function(options) {
       // to be implemented by subclasses
     },
 
+    _updateFilterUrlParams: function() {
+      // get url parameters into an array
+      var params=[];
+      // if there are already parameters there, get them
+      if (window.location.href.split("?")[1]) {
+        params = $.deparam(window.location.href.split("?")[1]);
+      }
+      // get the filter settings
+      var filteringSettings = this.parent.table._getFilteringSettings();
+
+      // if there are active filters, put them in the ext_filter_json param
+      if (JSON.parse(filteringSettings).length>0) {
+        params.ext_filter_json = filteringSettings;
+      }
+      // otherwise delete the ext_filter_json param to keep a clean uri
+      else {
+        delete params.ext_filter_json;
+      }
+      // if history is supported, add it to the url
+      if (history.pushState) {
+        var state = { params: params };
+        var url =  window.location.href.split("?")[0] + "?" + jQuery.param(params);
+        history.pushState(state, window.document.title, url);
```

Comment:
Thanks for bringing up Vicente. For things that change as often as filters do.. I think replaceState makes more sense?  No one wants to have to hit back 6 times to get to summary view if they played with the filters a bit (IMO).

Can we defer to what we decided and has seemed to work well for a few years on the existing reports for filter changes?  Please check and confirm it matches behavior there